ts/boot/configureStoreAndPersistor.ts

```diff
@@ -106,8 +106,27 @@ const migrations: MigrationManifest = {
   },
 
   // Version 5
+  // we removed selectedFiscalCodes from organizations
+  "5": (state: PersistedState) => {
```

Comment:
Since on master the store is on version **5**, if you want to be sure the migration will be applied, this should be **_6_**

Reply:
I miss it always, I'll take more care of it! Updated within https://github.com/teamdigitale/io-app/pull/1260/commits/edcbb64642d5c4ac8a8256db8d4ffd0fb43d6226

Reply:
:wink: